playbooks/edx_sandbox.yml

```diff
@@ -42,6 +44,7 @@
       - xqueue
       nginx_default_sites:
       - lms
+    - aws
```

Comment:
@Metfriet, this shouldn't be present. The lines above in the change above this one, conditionally enable the `aws` role.